vividus-engine/src/main/java/org/vividus/resource/TestResourceLoader.java

```diff
@@ -54,9 +54,18 @@ public TestResourceLoader(IPropertyParser propertyParser, ResourcePatternResolve
         try
         {
             String normalizedResourceLocation = StringUtils.appendIfMissing(resourceLocation, SEPARATOR);
-            String locationPattern = resourceLocation.startsWith(FILE_URL_PREFIX)
-                    ? normalizedResourceLocation : CLASSPATH_ALL_URL_PREFIX + normalizedResourceLocation;
-            Resource[] allResources = resourcePatternResolver.getResources(locationPattern + "**/" + resourcePattern);
+            String locationPattern = null;
+            Resource[] allResources = null;
+            if (resourceLocation.startsWith(FILE_URL_PREFIX))
+            {
+                locationPattern = normalizedResourceLocation;
+                allResources = resourcePatternResolver.getResources(locationPattern + resourcePattern);
+            }
+            else
+            {
+                locationPattern = CLASSPATH_ALL_URL_PREFIX + normalizedResourceLocation;
+                allResources = resourcePatternResolver.getResources(locationPattern + "**/" + resourcePattern);
```

Comment:
@valfirst as I see in default behavior we add **/, but should not the user be responsible for adding this into resource pattern on his own?

Reply:
tricky question, but "backward compatibility" is our goal